beacon-chain/db/testing/cleanup_db.go

```diff
@@ -0,0 +1,17 @@
+package testing
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/db"
+)
+
+// Cleanup clears and closes a database.
+func Cleanup(t *testing.T, database db.Database) {
+	if err := database.ClearDB(); err != nil {
+		t.Error(err)
+	}
+	if err := database.Close(); err != nil {
+		t.Error(err)
+	}
+}
```

Comment:
Consider updating `t.Cleanup` method in `beacon-chain/db/testing/setup_db` (if you think that procedure there doesn't clean the DB properly). There is no reason to re-introduce cleanup method that manually will be defer called (one can always limit scope using `r.Run` and within that scope call `t.SetupDB()` which will register cleanup handler).